java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -1104,7 +1104,7 @@ void testWindowStatic() {
       }
 
       try (ColumnVector expected = ColumnVector.fromInts(2, 3, 3, 3, 2);
-           ColumnVector result = v1.rollingWindow(AggregateOp.COUNT, options)) {
+           ColumnVector result = v1.rollingWindow(AggregateOp.COUNT_VALID, options)) {
```

Comment:
Pinging @mythrocks for visibility. I do wonder if we want to wrap the ops for rolling window, if that makes sense... so we don't depend on the enum outside of jni cudf? That said @nartal1, I don't think that's a change for this PR.

Reply:
I had to update it since we don't have `COUNT` anymore.  Based on @razajafri 's comment I was planning to add a new test something like below. Please let me know how to go about this.
`
 @Test
  void testWindowStaticCounts() {
    WindowOptions options = WindowOptions.builder().window(2, 1)
            .minPeriods(2).build();
    try (ColumnVector v1 = ColumnVector.fromBoxedInts(5, 4, null, 6, 8)) {
      try (ColumnVector expected = ColumnVector.fromInts(2, 2, 2, 2, 2);
           ColumnVector result = v1.rollingWindow(AggregateOp.COUNT_VALID, options)) {
        assertColumnsAreEqual(expected, result);
      }
      try (ColumnVector expected = ColumnVector.fromInts(2, 3, 3, 3, 2);
           ColumnVector result = v1.rollingWindow(AggregateOp.COUNT_ALL, options)) {
        assertColumnsAreEqual(expected, result);
      }
    }
  }`

Reply:
You mean like a separate `WindowAggregateOp` type? That would still depend on `cudf::experimental::aggregate::Kind`, at the moment. :/

Reply:
@mythrocks ok if it is not going to lend itself to a similar interface from the group by aggregate, that is fine. I was mostly curious.